syntax = "proto3";

import "matrix.proto";

package bayesmix;

message UniLSState {
  double mean = 1;
  double var = 2;
}

message MultiLSState {
  Vector mean = 1;
  Matrix prec = 2;
}

message LinRegUniLSState {
  Vector regression_coeffs = 1;
  double var = 2;
}
